pyfolio/tears.py

```diff
@@ -1513,19 +1530,56 @@ def create_perf_attrib_tear_sheet(returns,
     perf_attrib.show_perf_attrib_stats(returns, positions, factor_returns,
                                        factor_loadings, transactions)
 
-    vertical_sections = 3
+    vertical_sections = 1 + 2 * len(factor_partitions)
```

Comment:
Mind just adding a comment here that indicates what different sections are? I think it would make it clearer what we want the layout to look like from just reading the code

Reply:
added a clarifying comment